app/src/main/res/layout/home_fragment.xml

```diff
@@ -14,7 +14,7 @@
   <FrameLayout
```

Comment:
In `home_activity.xml` use `ToolbarTextAppearance` instead `ToolbarTitle` for `titleTextAppearance` so as to keep it consistent with other code.

Reply:
So can we use `ToolbarTextAppearance ` in other file also such as `profile_progress_activity` also using the same `ToolbarTitle `.
Also there are other files also. So what you suggest @rt4914 ?

Reply:
Doing it for `profile_progress_activity ` and `home_activity ` is fine for now. For other activities in document mention that we have to remove `ToolbarTitle` post PR changes

Reply:
Okay, I already done this for `profile_progress_activity `. 
And I'll add it to Post PR changes.